src/main/java/org/spongepowered/common/config/category/ModuleCategory.java

```diff
@@ -30,32 +30,33 @@
 @ConfigSerializable
 public class ModuleCategory extends ConfigCategory {
 
-    @Setting(value = "bungeecord")
+    @Setting(value = "bungeecord", comment = "???")
     private boolean pluginBungeeCord = false;
 
-    @Setting(value = "entity-activation-range")
+    @Setting(value = "entity-activation-range", comment = "???")
     private boolean pluginEntityActivation = true;
 
-    @Setting(value = "tileentity-activation", comment = "Controls block range and tick rate of tileentities. \n"
-                                                      + "Use with caution as this can break intended functionality.")
+    @Setting(value = "tileentity-activation", comment = ""
+            + "Controls block range and tick rate of tileentities.\n"
+            + "Use with caution as this can break intended functionality.")
     private boolean pluginTileEntityActivation = false;
 
-    @Setting(value = "entity-collisions")
+    @Setting(value = "entity-collisions", comment = "???")
     private boolean pluginEntityCollisions = true;
 
-    @Setting("timings")
+    @Setting(value = "timings", comment = "???")
```

Comment:
Module to run Aikar's Timings, profile your server performance and get a sharable web result https://github.com/aikar/timings#aikars-minecraft-timings-viewer-v2